qa/L0_perf_nomodel/run_test.sh

```diff
@@ -42,26 +41,38 @@ TENSOR_SIZE=${TENSOR_SIZE:=1}
 SHARED_MEMORY=${SHARED_MEMORY:="none"}
 REPORTER=../common/reporter.py
 
-DATADIR=/data/inferenceserver/${REPO_VERSION}
+DATADIR=${DATADIR:="/data/inferenceserver/${REPO_VERSION}"}
 RESULTDIR=${RESULTDIR:=.}
 
-SERVER=/opt/tritonserver/bin/tritonserver
-SERVER_ARGS="--model-repository=`pwd`/models"
+OPTDIR=${OPTDIR:="/opt"}
+SERVER=${OPTDIR}/tritonserver/bin/tritonserver
+BACKEND_DIR=${OPTDIR}/tritonserver/backends
+SERVER_ARGS="--model-repository=`pwd`/models --backend-directory=${BACKEND_DIR}" 
 source ../common/util.sh
 
+if [ "$TEST_JETSON" -eq 1 ]; then
+    ARCH="aarch64"
+    PERF_CLIENT=${OPTDIR}/tritonserver/clients/bin/perf_client
+else
+    ARCH="x86_64"
+    PERF_CLIENT=../clients/perf_client
```

Comment:
Personally I would make these variables (`ARCH` / `PERF_CLIENT`) to be configurable like `DATADIR`, and transfer the setting to top level where the test is configured in CI. So the script doesn't tie with a particular setup, think of adding yet another environment for testing, currently you will need to modify the `test.sh` / `run_test.h` / CI script to add the conditional, while it can just be changes in CI script (knowing what environment it is running against and set the variables at the top level)

Reply:
I think PERF_CLIENT (just like server) should be configured based on TRITON_DIR. 
I did make the changes for ARCH and got rid of TEST_JETSON

Reply:
Make sure you maintain the consistency in all L0_* tests running on jetson.